src/mochawesome.js

```diff
@@ -125,74 +116,24 @@ function Mochawesome(runner, options) {
 
   let endCalled = false;
 
-  // Add a unique identifier to each suite/test/hook
```

Comment:
Did you intend to remove this block? It is still needed by the reporter.

Reply:
@adamgruber, you're right, I have returned it back.
For some reason, I thought it was mine when I did parallel support.